src/tools/auth0/handlers/rules.ts

```diff
@@ -182,33 +201,47 @@ export default class RulesHandler extends DefaultHandler {
     // Do nothing if not set
     if (!rules) return;
 
-    // Figure out what needs to be updated vs created
-    const changes = await this.calcChanges(assets);
-
-    // Temporally re-order rules with conflicting ordering
-    await this.client.pool
-      .addEachTask({
-        data: changes.reOrder,
-        generator: (rule) =>
-          this.client.rules
-            .update({ id: rule.id }, stripFields(rule, this.stripUpdateFields))
-            .then(() => {
-              const updated = {
-                name: rule.name,
-                stage: rule.stage,
-                order: rule.order,
-                id: rule.id,
-              };
-              log.info(`Temporally re-order Rule ${convertJsonToString(updated)}`);
-            }),
-      })
-      .promise();
-
-    await super.processChanges(assets, {
-      del: changes.del,
-      create: changes.create,
-      update: changes.update,
-      conflicts: changes.conflicts,
-    });
+    log.warn(
+      'Rules are deprecated, migrate to using actions instead. See: https://auth0.com/docs/customize/actions/migrate/migrate-from-rules-to-actions for more information.'
+    );
+
+    try {
+      // Figure out what needs to be updated vs created
+      const changes = await this.calcChanges(assets);
+
+      // Temporally re-order rules with conflicting ordering
+      await this.client.pool
+        .addEachTask({
+          data: changes.reOrder,
+          generator: (rule) =>
+            this.client.rules
+              .update({ id: rule.id }, stripFields(rule, this.stripUpdateFields))
+              .then(() => {
+                const updated = {
+                  name: rule.name,
+                  stage: rule.stage,
+                  order: rule.order,
+                  id: rule.id,
+                };
+                log.info(`Temporally re-order Rule ${convertJsonToString(updated)}`);
+              }),
+        })
+        .promise();
+
+      await super.processChanges(assets, {
+        del: changes.del,
+        create: changes.create,
+        update: changes.update,
+        conflicts: changes.conflicts,
+      });
+    } catch (err) {
+      if (isDeprecatedError(err)) {
+        log.warn(
+          'Failed to update rules not because functionality has been deprecated in favor of actions. See: https://auth0.com/docs/customize/actions/migrate/migrate-from-rules-to-actions for more information.'
```

Comment:
```suggestion
          'Failed to update rules because functionality has been deprecated in favor of actions. See: https://auth0.com/docs/customize/actions/migrate/migrate-from-rules-to-actions for more information.'
```